### ISSUE TITLE
Error: cannot find module 'aproba'

### ISSUE THREAD

**Issue by LucasCtrl** [npm/npm#18966]
I have a problem with npm for two days now. When I run a command containing `npm`, there is this error.
![image](https://user-images.githubusercontent.com/19686996/32047020-ffd3aeae-ba44-11e7-8086-5942f16ff60b.png)

I tried to uninstall node and npm and nothing to do, always the same problem

**Comment by rodcunha** [comment]
I've just updated with **npm i -g npm** and have the same problem.  Using a windows PC and tried uninstalling and re-installing node with no success.
I've copied the contents of the node_modules directory to the AppData/Roaming/npm/node_modules but I'm still getting this error:

Error: Cannot find module 'proto-list'

**Comment by rodcunha** [comment]
Ok Fixed it by removing the node_modules directory completely and running 

**npm install** 

once I've done this, I ran **npm i -g npm** and it restored to the previous version. babel and other modules seem to be running as intended. 

Hope it helps.

**Comment by neetudas** [comment]
Same error but removing npm_modules didn't helped. Any other method ?🙂

**Comment by zcmgyu** [comment]
I just install latest NodeJS. 
https://nodejs.org/en/

**Comment by JimKay1941** [comment]
I just installed this version: node-v8.9.1-x64.msi

But that did not help at all.

**Comment by johnstsai** [comment]
latest NodeJS helped in my case

**Comment by JimKay1941** [comment]
The single command I want to enter is: 

npm profile enable-2fa

This command is supported only in an npm release LATER than the one included with the latest release of NodeJS. I have installed the latest NodeJS and the npm there does not support that command.

**Comment by pille-ip** [comment]
Win10 home 64bit. After npm i -g npm started getting this weird error message. 
1. uninstall and reinstall of latest node js per 64bit msi windows installer did not work. 
2. removing node-modules in install folder and using the repair function of the latest msi windows installer did not work
3. Uninstall and install of the latest LTE 64bit msi did not work. 
4. I removed both the node modules and cache folders under AppData/Roaming and ran the msi installer with the repair command. I used the LTE version 8.9.3 64bit for that. This worked for me.
